API/controllers/auth_controller.go

```
@@ -0,0 +1,39 @@
+	c.Request.URL.RawQuery = q.Encode()
+	
+	user, err := gothic.CompleteUserAuth(c.Writer, c.Request)
+	if err != nil {
+		c.JSON(http.StatusUnauthorized, gin.H{"error": "Erreur lors de l'authentification", "message": err.Error()})
+		return
+	}
+	c.JSON(http.StatusOK, gin.H{"user": user})
+}
+
+func Logout(c *gin.Context) {
+	gothic.Logout(c.Writer, c.Request)
```

golangci-lint (1.62.2)

Error return value of `gothic.Logout` is not checked

(errcheck)